src/Core/src/Platform/Windows/MauiSlider.cs

```diff
@@ -64,23 +52,13 @@ public WImageSource ThumbImageSource
 			set { SetValue(ThumbImageSourceProperty, value); }
 		}
 
-		internal event EventHandler Ready;
-
 		protected override void OnApplyTemplate()
```

Comment:
@mattleibow another tricky part about getting rid of `MauiSlider` is that the logic in `OnApplyTemplate` has to move into an event handler. In my experiments using the `Loaded` event seemed to work well for this, but I know little about the actual control lifecycle.